index.js

```diff
@@ -101,7 +102,8 @@ async function terraform(terraformDirectory, args) {
   }
 
   core.startGroup('Run terraform init');
-  const tfi = await terraform(terraformDirectory, ['init']);
+  let tfi_args = !terraformBackend ? ['init'] : ['init', '-backend-config=' + terraformBackend];
```

Comment:
Feels a bit clunky to have `init` there twice. But, all alternatives I can think of aren't that much better :thinking: